2015/11.md

```diff
@@ -73,10 +73,10 @@ Please register here (https://ecma-international.doodle.com/poll/fubwuawp6zu3q4y
   1. [Intl.PluralRules](https://github.com/caridy/intl-plural-rules-spec/blob/master/PROPOSAL.md) (Caridy Patiño & Eric Ferraiuolo)
   1. [Intl.RelativeTimeFormat](https://github.com/caridy/intl-relative-time-spec/blob/master/PROPOSAL.md) (Caridy Patiño & Eric Ferraiuolo)
 1.  Date and place of the next meetings
-2.  1/26-28/2015    San Francisco  Salesforce
-3.  3/s9-31/2015    San Francisco  Mozilla
-4.  5/24-26/2015    Munich, Ger    Google
-5.  7/26-29/2015    Remond         Microsoft
-6.  9/27-29/2015    Los Gatos      Netflix
-7.  11/29-12/1/2015 Menlo Park     Facebook
+2.  1/26-28/2016    San Francisco  Salesforce
+3.  3/s9-31/2016    San Francisco  Mozilla
```

Comment:
@ljharb Fixed in [this PR](https://github.com/tc39/agendas/pull/134), along with the four-day meeting in July and the ambiguous date formatting.